packages/plugin-constraints/sources/Constraints.ts

```diff
@@ -98,39 +98,47 @@ function extractError(val: any) {
 
 class Session {
   private readonly session: pl.type.Session;
+  public readonly main: Thread;
 
-  public constructor(project: Project, source: string) {
-    // The default Tau Prolog resolution steps limit is quite small
-    // By using a proportional limit (instead of no limit), we avoid triggering pathological cases "easily"
-    // See https://github.com/yarnpkg/berry/issues/1260
-    const limit = 1000 * project.workspaces.length;
-    this.session = pl.create(limit);
```

Comment:
Is this no longer true in the latest version?

Reply:
Nope, that change is unintended 🤔